iomgr/lib/dbengine/parser/AntlrHelpers.h

```diff
@@ -24,6 +24,14 @@ inline std::string& unquoteString(std::string& s)
 {
     s.pop_back();
     s.erase(0, 1);
+
+    size_t position = 0;
```

Comment:
Please move this new twice repeated piece of code into the separate function called `fixSingleQuotes`. You can place it right above this function. (DRY - Don't Repeat Yourself)

code style: or all namespace std members, even those which are for historical reasons still exposed into global namespace, we use `std::` prefix, so it must be `std::size_t` but not just `size_t`.